### PR TITLE
update from dev to doves expansion

### PR DESCRIPTION
**Pull request by butlerx**
it past card test

### CODE REVIEW

File: config/cards/DevOpsAgainstHumanityExpansion_answers.json [hunk 1]
```diff
@@ -6174,5 +6062,173 @@
...
+    "type": "Answer",
+    "value":"A Github Outage",
+    "keep": "Yes",
+    "draw": 0,
+    "pick": 1,
+    "source": "Dev Against Humanity Expansion"
+  },
+  {
+    "type": "Answer",
+    "value":"A Pi̱ata full of error messages",
```
Comment (creadak) [inline config/cards/DevOpsAgainstHumanityExpansion_answers.json]: Is this a typo?

File: config/cards/DevOpsAgainstHumanityExpansion_answers.json [hunk 2]
```diff
@@ -6174,5 +6062,173 @@
...
+    "type": "Answer",
+    "value": "3-hour standups",
+    "keep": "Yes",
+    "draw": 0,
+    "pick": 1,
+    "source": "Dev Against Humanity Expansion"
+  },
+  {
+    "type": "Answer",
+    "value": "A 3.5' floppy disk",
```
Comment (creadak) [inline config/cards/DevOpsAgainstHumanityExpansion_answers.json]: Any chance you could change the ' to \" ?

Comment (butlerx) [inline config/cards/DevOpsAgainstHumanityExpansion_answers.json]: appears to be a duplicate anyway

File: config/cards/DevOpsAgainstHumanityExpansion_answers.json (hunk 1 above)
Comment (butlerx) [inline config/cards/DevOpsAgainstHumanityExpansion_answers.json]: i assume its Piñata and have change it accordingly

**Pull request merged**